app/models/registry.rb

```diff
@@ -65,7 +65,9 @@ def find_image_by_name(name)
   private
 
   def endpoint_url_without_scheme
-    endpoint_url.gsub(/^#{parsed_endpoint_uri.scheme}:\/\//,'')
+    endpoint_url
+      .gsub(/^#{parsed_endpoint_uri.scheme}:\/\//,'')
+      .gsub(/\/+$/, '')
```

Comment:
we could compose this thing instead of using a regex to remove the scheme, but it'd mean we'd have to duplicate a lot of the code from: https://github.com/ruby/ruby/blob/trunk/lib/uri/generic.rb#L1341